src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -348,6 +406,48 @@ private void EnsureSignature()
             }
         }
 
+        /// <summary>
+        /// Ensures that binding is done so we guarantee that we have the <see cref="BackingField"/> or
```

Comment:
>Ensures that binding is done [](http://example.com/codeflow?start=12&length=28)

I do not think this is accurate. The purpose of this method is to guarantee that we created a backing field if that field is necessary. In some scenarios binding is not necessary even though the field is. Performing the binding is not the purpose of this method, however it might do the binding as a way to achieve the task. The purpose and the mechanism are two different things. So, in my opinion saying "Ensures that binding is done" is somewhat misleading and likely to confuse the reader.

Reply:
Updated the doc comment. Does it look better now?

Reply:
> Updated the doc comment. Does it look better now?

Looks better. Thanks.